lib/pbio/drv/legodev/legodev_pup.c

```diff
@@ -384,6 +376,35 @@ struct _pbdrv_legodev_dev_t {
 
 static pbdrv_legodev_dev_t devs[PBDRV_CONFIG_LEGODEV_PUP_NUM_INT_DEV + PBDRV_CONFIG_LEGODEV_PUP_NUM_EXT_DEV];
 
+PROCESS_THREAD(pbio_legodev_pup_process, ev, data) {
+    #if PBDRV_CONFIG_LEGODEV_PUP_POWER_CYCLE_PORTS
+    static struct etimer timer;
+    #endif
+
+    static int i;
+
+    PROCESS_BEGIN();
+
+    #if PBDRV_CONFIG_LEGODEV_PUP_POWER_CYCLE_PORTS
```

Comment:
I've been adding `QUIRK` to config options for hacks like this.

```suggestion
    #if PBDRV_CONFIG_LEGODEV_PUP_QUIRK_POWER_CYCLE_PORTS
```